price-feeder/oracle/oracle_test.go

```diff
@@ -697,3 +697,94 @@ func TestGetComputedPricesTickersConversion(t *testing.T) {
 		prices[btcPair.Base],
 	)
 }
+
+func TestGetComputedPricesPanic(t *testing.T) {
```

Comment:
Pls rename this to a description of the case rather than the behavior

Reply:
I assume this test will get deleted once the non temporary solution is implemented

Reply:
or we could just modify it to be another test and check for real expected values

Reply:
@RafilxTenfen maybe something like `TestGetComputedPricesEmptyTvwap`?

Reply:
done at https://github.com/umee-network/umee/pull/1262/commits/be876842a75dbc9d0c19a8351a233005dee3853a